js/repl/Repl.js

```diff
@@ -317,7 +329,11 @@ class Repl extends React.Component {
         sourceMap: runtimePolyfillState.isEnabled,
         useBuiltIns: state.builtIns,
       })
-      .then(result => this.setState(result, setStateCallback));
+      .then(result => {
+        result.meta.compiledSize = prettySize(result.meta.compiledSize);
+        result.meta.rawSize = prettySize(result.meta.rawSize);
```

Comment:
can do `{...result,  meta: {...result.meta, rawSize:0, compiledSize:0 }}` if some additional bytes won't matter by not mutating param ? :smile: